extensions/notebook/src/book/bookModel.ts

```diff
@@ -29,6 +29,7 @@ export class BookModel {
 	private _errorMessage: string;
 	private _activePromise: Deferred<void> | undefined = undefined;
 	private _queuedPromises: Deferred<void>[] = [];
+	private _rootNode: BookTreeItem;
```

Comment:
Please add a comment explaining what this "root node" is

Reply:
> You have a couple failing tests now

Just checked, those tests are failing in master as well. But looking into fixing them.